docs/architecture/posixfs.md

```diff
@@ -0,0 +1,173 @@
+---
+title: "Posix Filesystem"
+date: 2024-05-27T14:31:00+01:00
+weight: 30
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/architecture
+geekdocFilePath: posixfs.md
+---
+
+{{< toc >}}
+
+Posix FS is the working name for the collaborative storage driver for Infinite Scale.
+
+The scope of this document is to give an high level overview to the technical aspects of the Posix FS and guide the setup.
+
+## Introduction
+
+Posix FS is a backend component that manages files on the server utilizing a "real" file tree that represents the data with folders and files in the file system as users are used to it. That is the big difference compared to Decomposed FS which is the default storage driver in Infinite Scale.
+
+This does not mean that Infinte Scale is trading any of it's benefits to this new feature: It still implements simplicity by running without a database, it continues to store metadata in the file system and adds them transparently to chaches and search index, and it also features the full spaces concept as before, just to name a few example.
+
+The architecture of Infinite Scale allows to configure different storage drivers for specific storage types and purposes on a space granularity. The Posix FS storage driver is an alternative to the default driver called Decomposed FS.
+
+However, the clearance of the file structure in the underlying file system is not the only benefit of the Posix FS. This new technology allows users to manipulate the data directly in the file system, and any changes to files made aside of Infinite Scale are monitored and directly reflected in Infinite Scale.
+
+The first time ever with feature rich open source file synce & share systems, users can either choose to work with their data through the clients of the system, it's API's or even directly in the underlying file system on the server.
+
+That is another powerful vector for integration and enables a new spectrum of use cases across all domains. Just imagine how many software can write files, and can now directly make them accessible real time in a convenient, secure and efficient way.
```

Comment:
> Just imagine how many software can write files, and can now directly make them accessible real time in a convenient, secure and efficient way.

This I don't understand. Are you talking about other competitors who can't do this?